File: cv-backend/src/main/java/a/gleb/cv/model/request/ContactInfoModel.java
package a.gleb.cv.model.request;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class ContactInfoModel {
    @JsonProperty("phone")
    private String phoneNumber;
    @JsonProperty("email")
    private String email;
    @JsonProperty("tg")
    private String telegramLink;
    @JsonProperty("linkedIn")
    private String linkedInLink;
    @JsonProperty("facebook")
    private String faceBookLink;
    @JsonProperty("github")
    private String githubLink;
    @JsonProperty("gitlab")
    private String gitlabLink;
}

File: cv-frontend/lib/api/models/contact_info_model.dart
import 'package:dart_json_mapper/dart_json_mapper.dart';

@jsonSerializable
class ContactInfoModel {
  @JsonProperty(name: "phone")
  final String phoneNumber;
  @JsonProperty(name: "email")
  final String email;
  @JsonProperty(name: "tg")
  final String telegramLink;
  @JsonProperty(name: "linkedIn")
  final String linkedInLink;
  @JsonProperty(name: "facebook")
  final String faceBookLink;
  @JsonProperty(name: "github")
  final String githubLink;
  @JsonProperty(name: "gitlab")
  final String gitlabLink;

  ContactInfoModel(this.phoneNumber, this.email, this.telegramLink,
      this.linkedInLink, this.faceBookLink, this.githubLink, this.gitlabLink);

  @override
  String toString() {
    return '{'
        '"phone": "$phoneNumber",'
        '"email": "$email", '
        '"tg": "$telegramLink", '
        '"linkedIn": "$linkedInLink", '
        '"facebook": "$faceBookLink", '
        '"github": "$githubLink", '
        '"gitlab": "$gitlabLink"}';
  }
}
